dev/com.ibm.websphere.appserver.thirdparty.eclipselink.2.7/resources/org/eclipse/persistence/internal/localization/i18n/LoggingLocalizationResource.nlsprops

```diff
@@ -175,6 +175,18 @@ elements={0}{1} elements
 
 enable_batch_operations_as_true_not_supported=WLS native CMP setting "enable-batch-operations-as-true" is not directly supported in EclipseLink CMP. See migration doc for details.
 
+encryptor_decrypt_old_algorithm=Database password used in {0} was encrypted by deprecated algorithm.
+It is recommended to re-encrypt it by `passwordUpdate.sh` from eclipselink.zip bundle.
+
+encryptor_decrypt_old_algorithm_without_session_name=Database password was encrypted by deprecated algorithm.
+It is recommended to re-encrypt it by `passwordUpdate.sh` from eclipselink.zip bundle.
+
+encryptor_script_description=This application internally decrypt old encrypted password used by some previous version EclipseLink and encrypt it by latest algorithm.
+
+encryptor_script_output=Re-encrypted password is: {0}
```

Comment:
I personally prefer "Re-encrypted" but current spelling is "Reencrypted" (no hyphen/dash)